### ISSUE TITLE
Doesn't work on quarantined subreddits.

### ISSUE THREAD

**Issue by XLR-XR** [ostrolucky/Simple-Subreddit-Image-Downloader#5]
Quarantined subreddits, such as /r/smuggies, return 0 image results.

**Comment by ostrolucky** [comment]
Indeed. I don't see good way to solve this though, so this will be won't fix unless somebody steps up to fix this.

**Comment by ostrolucky** [comment]
Let's leave the issue open to show that I'm open to contribution for fixing this
